lia-ui/src/main/java/com/lithium/community/android/ui/components/adapters/LiConversationAdapter.java

```diff
@@ -185,9 +185,7 @@ private void showPopupMenu(final View view, final int position) {
             popup.getMenu().removeItem(R.id.li_action_delete);
         }
 
-        boolean showSubscribe = (position == 0);
         boolean isSubscribed = item.getUserContext().isSubscribed();
```

Comment:
I think this change was part of another commit.. so should not be part of this after rebasing

Reply:
github didn't resolve, merge and rebase. But it is not conflicting. I think should be alright, ...?